contracts/okp4-dataverse/src/msg.rs

```diff
@@ -120,6 +123,19 @@ pub enum ExecuteMsg {
     },
 }
 
+/// # TripleStoreConfig
+/// `TripleStoreConfig` represents the configuration related to the management of the triple store.
+#[cw_serde]
+pub struct TripleStoreConfig {
+    /// The code id that will be used to instantiate the triple store contract in which
+    /// to store dataverse semantic data. It must implement the cognitarium interface.
+    pub code_id: Uint64,
+
+    /// Limitations regarding triple store usage.
+    #[serde(default = "okp4_cognitarium::msg::StoreLimitsInput::default")]
+    pub limits: okp4_cognitarium::msg::StoreLimitsInput,
```

Comment:
I would use a custom type specific to this smart contract to make uncoupled at interface level. Because if there is a change in `StoreLimitsInput` (for example a new property), it will affect this contract, without any warning.